File: One Time Pad Library/src/KeyReader.java
import java.io.*;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.BitSet;

/**
 * Created by Oliver on 17/05/2018.
 * Written by Oliver Bathurst <oliverbathurst12345@gmail.com>
 */

class KeyReader {

    KeyReader(){}

    VernamKey readKey(String filePath) throws IOException {
        return new VernamKey(BitSet.valueOf(Files.readAllBytes(Paths.get(filePath))));
    }

    byte[] readKeyIntoByteArray(String filePath) throws IOException {
        return Files.readAllBytes(Paths.get(filePath));
    }

    BitSet readKeyIntoBitSet(String filePath) throws IOException {
        return BitSet.valueOf(Files.readAllBytes(Paths.get(filePath)));
    }
}

File: CSharp version/One Time Pad/One Time Pad CS/KeyReader.cs
using System;
using System.Collections;
using static System.IO.File;
using static System.IO.Path;

class KeyReader{

    public KeyReader() {}

    public VernamKey readKey(String filePath){
        return new VernamKey(new BitArray(ReadAllBytes(GetFullPath(filePath))));
    }

    public byte[] readKeyIntoByteArray(String filePath){
        return ReadAllBytes(GetFullPath(filePath));
    }

    public BitArray readKeyIntoBitArray(String filePath){
        return new BitArray(ReadAllBytes(GetFullPath(filePath)));
    }
}
